### DISCUSSION TITLE
API Query operators on text fields: starts_with and ends_with

### DISCUSSION THREAD

**Discussion by qroac** [payloadcms/payload#4573] [Feature Requests & Ideas]
I am working on some filter functions for huge data tables in our frontend. 
For some of the intended functions, I need filter operations that I cannot find in the [documentation](https://payloadcms.com/docs/queries/overview#querying-your-documents):

`starts_with` and `ends_with` to search for records that have the queried field starting or ending with a required value

Is this already possible and just not listed in docs?
Or is this already planned for future releases?

**Comment by materazu** [comment]
Up, we need this one on our project.
